mmv1/products/transcoder/JobTemplate.yaml

```diff
@@ -0,0 +1,947 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "JobTemplate"
+self_link: "projects/{{project}}/locations/{{location}}/jobTemplates/{{job_template_id}}"
+base_url: "projects/{{project}}/locations/{{location}}/jobTemplates"
+create_url: "projects/{{project}}/locations/{{location}}/jobTemplates?jobTemplateId={{job_template_id}}"
+update_verb: :PATCH
+update_mask: true
+description: "Transcoding Job Template Resource"
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Transcoder": "https://cloud.google.com/transcoder/docs/"
+  api: "https://cloud.google.com/transcoder/docs/reference/rest/v1/projects.locations.jobTemplates"
+import_format: ["projects/{{project}}/locations/{{location}}/jobTemplates/{{job_template_id}}"]
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "transcoder_job_template_basic"
+    primary_resource_id: "default"
+    vars:
+      job_template_id: "example-job-template"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "jobTemplateId"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      ID to use for the Transcoding job template.
+  - !ruby/object:Api::Type::String
+    name: "location"
+    immutable: true
+    required: true
+    url_param_only: true
+    description: |
+      The location of the transcoding job template resource.
+properties:
+  - !ruby/object:Api::Type::String
+    name: "name"
```

Comment:
since we're using `job_template_id` over `name` we can mark it as output only.

Reply:
fixed in 3bfcb6f